src/cnxml/util.js

```diff
@@ -401,3 +401,12 @@ export const DEFAULT = {
         return mixedContent(el, next)
     },
 }
+
+export const cleanAttrs = obj => {
+    for (const key in obj) {
+        if (obj[key] === null || obj[key] === undefined || obj[key] === '') {
```

Comment:
An empty string is a valid value (it's for example the default value for `item-sep`) so it shouldn't be filtered out, and remaining checks can be collapsed into `obj[key] == null`.